python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -718,13 +715,47 @@ def process_graph(args):
                 else:
                     g.edges[etype].data[name] = th.tensor(edata)
         dgl.save_graphs(os.path.join(args.output_dir, args.graph_name + ".dgl"), [g])
+        output_dirs = [args.output_dir]
+    elif args.output_format == "both":
+        # Save both DistDGL and DGL graph
+        for ntype in node_data:
+            for name, ndata in node_data[ntype].items():
+                if isinstance(ndata, ExtMemArrayWrapper):
+                    g.nodes[ntype].data[name] = ndata.to_tensor()
+                else:
+                    g.nodes[ntype].data[name] = th.tensor(ndata)
+        for etype in edge_data:
+            for name, edata in edge_data[etype].items():
+                if isinstance(edata, ExtMemArrayWrapper):
+                    g.edges[etype].data[name] = edata.to_tensor()
+                else:
+                    g.edges[etype].data[name] = th.tensor(edata)
+        dglgraph_output_dir = os.path.join(args.output_dir, "dglgraph")
+        dgl.save_graphs(os.path.join(dglgraph_output_dir, args.graph_name + ".dgl"), [g])
+
+        assert args.part_method in ["metis", "random"], \
+                "We only support 'metis' or 'random'."
+
+        distdgl_output_dir = os.path.join(args.output_dir, "dist_dglgraph")
+        partition_graph(g, node_data, edge_data, args.graph_name,
+                        args.num_parts, distdgl_output_dir,
+                        save_mapping=True, # always save mapping
+                        part_method=args.part_method)
+        output_dirs = [dglgraph_output_dir, distdgl_output_dir]
```

Comment:
do we have to save them in separate folders? how about saving them in the same folder?

Reply:
DGL format has only one file.

Reply:
We can